arch/cpu/arm/cortex-m/trustzone/normal/Makefile.normal

```diff
@@ -0,0 +1,2 @@
+# Exclude the normal dbg-arch.c from the build
+CONTIKI_SOURCES_EXCLUDES += dbg-arch.c
```

Comment:
Instead of first adding and then removing the file, wrap the place that adds this file with a `ifndef TRUSTZONE_SECURE_BUILD`.